VocaDb.Migrations/Migrations.cs

```diff
@@ -5,15 +5,96 @@ namespace VocaDb.Migrations {
 
 	// Migration version format: YYYY_MM_DD_HHmm
 
+	[Migration(2020_03_01_1300)]
+	public class ArchivedEventVersionsRenameVenue : Migration {
+
+		public override void Up() {
+			Execute.Sql("UPDATE [ArchivedEventVersions] SET ChangedFields = REPLACE(ChangedFields, 'Venue', 'VenueName') WHERE ChangedFields LIKE '%Venue%' AND NOT ChangedFields LIKE '%VenueName%'");
+		}
+
+		public override void Down() {
+			Execute.Sql("UPDATE [ArchivedEventVersions] SET ChangedFields = REPLACE(ChangedFields, 'VenueName', 'Venue') WHERE ChangedFields LIKE '%VenueName%'");
+		}

```

Comment:
If the `Down` method is executed, we may not be able to distinguish `Venue` and `VenueName` anymore, because both of them become `Venue`.

Reply:
Well, you could rename Venue to VenueEntry first. But I don't think it's that important.